lib/rules.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class Rules
+  attr_reader :board
+
+  def initialize(board)
+    @board = board
+  end
+
+  def tie_end?
```

Comment:
I did not see a test for this method? If this is what determines a tie then the test currently being used to test for a tie is testing the wrong method